File: Events-App/Events.Web/Models/IndexModel.cs
using Events.Data.Entities;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Events.Web.Models
{
    public class IndexModel
    {
        public int Page { get; set; }
        public DateTime StartDate { get; set; }
        public IEnumerable<Event> Events { get; set; }
    }
}

File: Events-App/Events.Web/Bindings/Models/IndexModel.js
(function (ev) {
	let IndexModel = function (page, events) {

		let self = this;

		self.Events = ko.observableArray(events);
		self.Page = ko.observable(page);
		self.Date = ko.observable();
	};
	ev.IndexModel = IndexModel;
}(window.Events));
